pkg/mcs/exportspec.go

```diff
@@ -0,0 +1,161 @@
+package mcs
+
+import (
+	"bytes"
+	"encoding/json"
+	"os"
+	"reflect"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	mcsv1a1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+)
+
+// ExprtSpec holds the relevant data for creating imports and arbitrating conflicts
+// in service definitions.
+type ExportSpec struct {
+	// CreatedAt is the API master assigned creation time for the ServiceExport, used in
+	// conflict resolution.
+	CreatedAt metav1.Time `json:"createdAt"`
+	// ClusterID indicates the origin cluster represented by the ServiceExport.
+	ClusterID string `json:"clusterID"`
```

Comment:
I'm aware of that. Explicitly wanted to not assume anything regarding anything lighthouse already does (otherwise we would need to merge multiple input sources into the same struct).
I'll add it to a comment so it does not seem an oversight